Source/ThirdParty/Recast/Source/RecastMeshDetail.cpp

```diff
@@ -653,8 +653,8 @@ static bool buildPolyDetail(rcContext* ctx, const float* in, const int nin,
 	for (int i = 0; i < nin; ++i)
 		rcVcopy(&verts[i*3], &in[i*3]);
 	
-	edges.resize(0);
-	tris.resize(0);
+	edges.clear();
+	tris.clear();
```

Comment:
I wonder why they did this all over the place. `clear()` frees backing storage so memory reallocation will happen, while `resize(0)` keeps backing storage which is reused when new stuff is added to the container. I somewhat doubt there were memory usage issues...

Reply:
> clear() frees backing storage so memory reallocation will happen

Does it?

Reply:
How it's all related to storage?

Reply:
Eh... I mistakenly assumed this is backed by `std::vector<>`. Their custom containers do not free underlying memory in both cases. That explains the change